### PR TITLE
Feature/styling

### PR DESCRIPTION
**Pull request by vechicin**
# In this feature I did the following:

- Added styling to the page
- Update README screenshot to reflect styling

### CODE REVIEW

File: src/components/BookContainer/BookContainer.js [hunk 1]
```diff
@@ -6,9 +6,9 @@ import Book from '../Book/Book';
...
 const BookContainer = ({ books }) => (
   <section>
-    <ul>
+    <ul className="booklist">
       {books.map((book) => (
-        <li key={book.item_id}>
+        <li className="book" key={book.item_id}>
           <Book book={book} />
         </li>
       ))}
```
Comment (DeliceLydia) [inline src/components/BookContainer/BookContainer.js]: - [ ] If you can check the design they are some features that you did not implement yet, you should add them as a placeholder, you can refer to this image
cmc

File: src/components/navbar/Navbar.js [hunk 2]
```diff
@@ -3,13 +3,13 @@ import { Link } from 'react-router-dom';
...
       <li>
-        <Link to="/">Books</Link>
+        <Link to="/">BOOKS</Link>
       </li>
       <li>
-        <Link to="/categories">Categories</Link>
+        <Link id="categories-link" to="/categories">CATEGORIES</Link>
       </li>
     </ul>
   </nav>
```
Comment (DeliceLydia) [inline src/components/navbar/Navbar.js:15]: - [ ] In the navbar you should add an icon on your right

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @vechicin, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Major highlights for your project:
- Followed git workflow
- No linter errors
- README is awesome


## Required Changes ♻️
_Check the comments under the review._




### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Santi, 
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Please leave any questions or comments in the PR if you have any doubts or unclarity!
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**